src/asmcnc/keyboard/custom_keyboard.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+import sys
+from kivy.core.window import Window
+from kivy.uix.vkeyboard import VKeyboard
+
+try:
+    import hgtk
+except:
+    pass
+
+
+class Keyboard(VKeyboard):
+    def __init__(self, text_inputs, **kwargs):
+        super(Keyboard, self).__init__(**kwargs)
+
+        self.l = kwargs['localization']
+
+        # For python 2 encoding issues
+        reload(sys)
+        sys.setdefaultencoding('utf-8')
+
+        self.text_instance = None
+
+        self.kr_layout = "./asmcnc/keyboard/layouts/kr.json"
+        self.numeric_layout = "./asmcnc/keyboard/layouts/numeric.json"
+        self.qwerty_layout = "data/keyboards/qwerty.json"
```

Comment:
@VladPavelescu

Reply:
The path refers to the root Kivy installation because it is the default Kivy qwerty layout. As long as Kivy is installed on the device I think this should be fine (although I haven't tested it properly). You can find this in your `%PYTHON_INSTALLATION_PATH%/Lib/site-packages/kivy/data`.

Reply:
Ok. I don't understand why this works, but it doesn't seem to cause any issues 😅